source/stylesheets/components/_article.scss

```diff
@@ -4,4 +4,43 @@ article {
   @include media($large-screen) {
     @include span-columns(8.5);
   }
+
+  footer {
+    @include clearfix;
+    border-top: $base-border;
+    margin-top: 5em;
+    padding: $base-spacing 0;
+
+    .previous-guide {
+      float: left;
+
+      &::before {
+        content: "‹";
```

Comment:
I've been using plain characters thinking it was clearer, but I hadn't even considered fonts in editors. Great point. Fixed :+1: